### DISCUSSION TITLE
使用add_requires时不成功

### DISCUSSION THREAD

**Discussion by StarryPortrait** [xmake-io/xmake#6025] [Q&A] [answered]
使用add_requires时不成功，提示：`updating repositories .. error: fatal: not a git repository (or any of the parent directories): .git`。
我尝试使用 `git init`，没有效果。请问我该咋整

**Comment by waruqi** [comment, marked as answer]
```
xrepo rm-repo --all
```

**Reply by StarryPortrait** [reply]
谢谢大佬，行了
